concourse/enumerator.py

```diff
@@ -24,20 +24,36 @@
 from github3.exceptions import NotFoundError
 
 from ci.util import (
-    parse_yaml_file,
-    load_yaml,
-    info,
+    LintingError,
     fail,
-    verbose,
+    info,
+    lint_yaml,
+    load_yaml,
     merge_dicts,
     not_empty,
     not_none,
+    parse_yaml_file,
+    warning,
+    verbose,
 )
 from model.base import ModelBase, NamedModelElement
 from concourse.factory import RawPipelineDefinitionDescriptor
 import ccc.github
 
 
+REPLICATION_LINTING_CFG = {
+    'extends': 'relaxed',
+    'rules': {
+        'key-duplicates': {
+            'level': 'error',
+        },
+        'new-line-at-end-of-file': {
+            'level': 'warning',
```

Comment:
if you remove this relaxation, we could use the default profile, which I would prefer tbh